definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -952,6 +952,24 @@
     "EndButtonLabel": "Save and continue",
     "Publish": "True"
   },
+  {
+    "LiveFrom": "03/02/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "transferToAnotherCourt",
+    "Name": "Transfer to another court",
+    "Description": "Transfer to another court",
+    "DisplayOrder": 107,
```

Comment:
can we also hide the amend court event with the same PR

Reply:
Hided Amend court event